games-strategy/maxr/maxr-0.2.9.ebuild

```diff
@@ -0,0 +1,42 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=5
+inherit cmake-utils eutils
+
+DESCRIPTION="Mechanized Assault and Exploration Reloaded"
+HOMEPAGE="http://www.maxr.org/"
+SRC_URI="http://www.maxr.org/downloads/${P}.tar.gz"
+
+LICENSE="GPL-2 FDL-1.2+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+RESTRICT="mirror"
+IUSE="dedicated"
+
+RDEPEND=">=sys-devel/gcc-4.6:*
+	media-libs/libsdl2[video]
```

Comment:
I know the current ebuild doesn't have to but I suspect the `opengl` flag should be added here.

Reply:
Its pure 2d rendering, I'm not sure that opengl is needed. Binairy isn't linked with libGL.so., I checked with other software using opengl with sdl2 and they are linked with libGL.so

Reply:
Fair enough! I hadn't seen the screenshots until now.